python/invoke_release/tasks.py

```diff
@@ -1503,3 +1512,22 @@ def wheel(_):
         archive_name=archive_name,
         base_dir=base_dir
     ))
+
+
+def create_pull_request(options, user, token, title,
```

Comment:
Can you format this as a one-line? `def create_pull_request(options, user, token, title):`